src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/PackageFeeds/MultiSourcePackageMetadataProvider.cs

```diff
@@ -255,15 +257,24 @@ private async Task<IPackageSearchMetadata> GetPackageMetadataAsync(PackageIdenti
                 tasks.Add(_localRepository.GetPackageMetadataFromLocalSourceAsync(identity, cancellationToken));
             }
 
+            if (_globalLocalRepositories != null && _globalLocalRepositories.Any())
```

Comment:
The Any is redundant. Could use conditional access `?.` as well.

```suggestion
            if (_globalLocalRepositories != null)
```